server/src/MergeRequestCheckerLoop.ts

```diff
@@ -71,16 +72,29 @@ export class MergeRequestCheckerLoop {
 	private async task() {
 		console.log('[loop] Checking assigned merge requests');
 		const assignedMergeRequests = await this.gitlabApi.getAssignedOpenedMergeRequests();
-		const possibleToAcceptMergeRequests = assignedMergeRequests.map(
-			(mergeRequest: MergeRequest) =>
+		const possibleToAcceptMergeRequests = assignedMergeRequests
+			.filter((mergeRequest: MergeRequest): boolean => {
```

Comment:
Just a stylistic note: it is not common to explicitly specify parameter and return type of closures that TS can infer (such as in case of filter or map). In some circumstances explicitly specifying types may even reduce type safety (because the inferred types will be more precise). Here it is fine.